src/app/legacy/containers/PodcastExternalLinks/index.test.jsx

```diff
@@ -111,7 +119,7 @@ describe('Event Tracking', () => {
     render(<Component links={links} />);
 
     expect(clickTrackerSpy).toHaveBeenCalledWith({
-      componentName: 'third-party',
+      componentName: `third-party-${links[0].linkType}`,
```

Comment:
```suggestion
      componentName: 'third-party-${links[0].linkType}',
```

Reply:
Think this may error, the template literal is needed for `${links[0].linkType}`.